IABot/Parser/parse.php

```diff
@@ -34,6 +34,9 @@
 * @license https://www.gnu.org/licenses/gpl.txt
 * @copyright Copyright (c) 2016, Maximilian Doerr
 */
+require __DIR__ . '/../../vendor/autoload.php';
```

Comment:
I am unfamiliar with this file.  Why is it not where the other files are included?